README.md

```diff
@@ -117,3 +117,11 @@ You should see unlocked padlock icons on the right of every endpoint.
 4. Select 'Bearer Token' from the Auth Type dropdown list
 5. Paste the token into the box
 6. Postman will add the correct header
+
+## Using the vis.gl/react-google-maps API
+
+1. run npm install in the frontend folder
```

Comment:
This step is part of the general set up in the README so is not necessary to have here.

The information in the README should be aimed at someone who is newly downloading the app and doing their first set up.

The communication to devs who are already working on the app and will need to make changes like re-running `npm install` in the frontend after this PR has been merged should not be part of the README, it should happen as part of a message in general